src/status_im2/contexts/quo_preview/settings/accounts.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.quo-preview.settings.accounts
+  (:require [clojure.string :as string]
+            [quo2.components.settings.accounts.view :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as r]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Custom color"
+    :key     :custom-color
+    :type    :select
+    :options (mapv (fn [[k _]]
+                     {:key k, :value (string/capitalize (name k))})
+                   colors/customization)}
+   {:label "Account name"
+    :key   :account-name
+    :type  :text}
+   {:label "Account address"
+    :key   :account-address
+    :type  :text}])
+
+(defn cool-preview []
+  (let [state (r/atom {:custom-color    :blue
+                       :account-name    "Booze for Dubai"
+                       :account-address "0x21a ... 49e"
+                       ;; Fixed parameters:
+                       :avatar-icon     :main-icons/placeholder
+                       :on-press-menu   (fn []
+                                          (prn "menu pressed"))})]
```

Comment:
For fake event handler feedback, we've been using `(js/alert ...`), so designers using the preview also get feedback if something happened.

Reply:
It sounds good!
Solved in 5d66eb63d5418f2660063535946350b7d96ca840